packages/dd-trace/src/supported-configurations.json

```diff
@@ -365,6 +365,7 @@
     "DD_TRACE_MYSQL_ENABLED": ["A"],
     "DD_TRACE_MYSQL2_ENABLED": ["A"],
     "DD_TRACE_NATIVE_SPAN_EVENTS": ["A"],
+    "DD_TRACE_NATS_ENABLED": ["A"],
```

Comment:
This appears to be the only change in this file that has anything to do with this PR.

Reply:
explained over slack